server/src/main/scala/services/crunch/RunnableCrunch.scala

```diff
@@ -122,40 +126,39 @@ object RunnableCrunch {
           val liveSink = builder.add(Sink.actorRef(liveCrunchStateActor, "complete"))
           val fcstSink = builder.add(Sink.actorRef(fcstCrunchStateActor, "complete"))
           val aggregatedArrivalsSink = builder.add(Sink.actorRef(aggregatedArrivalsStateActor, "complete"))
+          val manifestsRequestSink = builder.add(Sink.actorRef(manifestsRequestActor, "complete"))
 
 
           baseArrivals ~> baseArrivalsFanOut ~> arrivals.in0
-                          baseArrivalsFanOut ~> baseArrivalsSink
+          baseArrivalsFanOut ~> baseArrivalsSink
```

Comment:
stick this indent back in, maybe there is some kind of directive to intellij that will stop reformatting